README.adoc

```diff
@@ -25,8 +25,14 @@ You'll explore how to manage the impact of failures using MicroProfile Fault Tol
 
 == What you'll learn
 
-You will learn how to build fault-tolerant microservices in an inventory management application so
-that you can reduce the impact from failure and ensure continued operation of services.
+You will learn how to use MicroProfile Fault Tolerance to build resilient microservices
+that reduce the impact from failure and ensure continued operation of services.
+
+MP Fault Tolerance provides a simple and flexible solution to build fault-tolerant microservices.
+Fault tolerance leverages different strategies to guide the execution and result of logic.
+Retry policies, bulkheads, and circuit breakers are popular concepts in this area.
+They dictate whether and when executions take place, and fallbacks offer an alternative result
+when an execution does not complete successfully.
```

Comment:
Much better and thank you.  The 2nd "Fault tolerance" should be "Fault Tolerance" still or we can use "it"?

Not sure why we tag on "and fallbacks offer an alternative result when an execution does not complete successfully" at the end.  Just feels a bit weird.  May be Sarah can help review.

Reply:
I think this looks good. I think the 2nd instance of "Fault Tolerance" is OK as it is.

Also, I'm not seeing a tag...was it already removed before I saw it?

Reply:
I think YK meant the sentence "and fallbacks offer an alternative result when an execution does not complete successfully" seems a bit weird in its place. Let me know if you have any suggestions @sishida Thanks.

Reply:
I think the 2nd "Fault tolerance" is needed to be clearer in this context.

Reply:
I specifically was referring to whether the "t" in "tolerance" should be capitalized for the 2nd "Fault tolerance".

Reply:
I agree with @sishida that the 2nd "Fault tolerance" is good, as it's not referring to the term any more, it's just "fault tolerance" in general, which fits in that context.

Reply:
Just checked, this sentence was taken directly from the spec..
"They dictate whether and when executions take place, and fallbacks offer an alternative result when an execution does not complete successfully."
Will work with @sishida to re-phase.

Reply:
Yes, we should rephrase.

Fwiw, "fault tolerance" in general doesn't do anything.  It is simply a concept so it can't really "leverages" anything.  If anything that will do anything, it is the MP Fault Tolerance implementation in OL in accordance to the spec.

Reply:
If we're speaking of "fault tolerance" as a property in general, then both words can be lower case. If we're referring to the name of a specific API, we can capitalize the words.

I didn't realize that "Retry policies, bulkheads, and circuit breakers are popular concepts in this area. They dictate whether and when executions take place, and fallbacks offer an alternative result when an execution does not complete successfully." was quoted directly from the spec. We can quote it directly since that is the case.